modules/light-clients/08-wasm/internal/ibcwasm/wasm.go

```diff
@@ -32,6 +34,16 @@ func GetVM() WasmEngine {
 	return vm
 }
 
+// SetQuerier sets the custom wasm query handle for the 08-wasm module.
+func SetQuerier(wasmQuerier WasmQuerier) {
+	querier = wasmQuerier
+}
+
+// GetQuerier returns the custom wasm query handler for the 08-wasm module.
+func GetQuerier() WasmQuerier {
+	return querier
```

Comment:
Would be nice to add a no-op (error only) implementation of `WasmQuerier` to return here `if querier == nil` - that way we avoid having a `nil` querier every time we call into the wasm vm.

Maybe an unexported private type could be added like `defaultQuerier` or `noopQuerier` which implements the interface, but returns zero for `GasConsumed` and `nil, errors.new("some informative error")` for `Query()`